tests/functional/geth/test_provider.py

```diff
@@ -72,14 +71,6 @@ def test_uri(geth_provider):
     assert geth_provider.uri == GETH_URI
 
 
-@geth_process_test
-def test_uri_localhost_not_running_uses_random_default(config):
```

Comment:
this now happens at the provider-connection level, so the config value is None when not actually configured (and doesn't get set to random RPC). So the behavior is the same, just happens at a different spot, it helps us know when a value was specifically configured by the user (and not injected here)